src/engine/BMGameAction.php

```diff
@@ -775,6 +775,20 @@ protected function message_recipe_change(array $preInfo, array $postInfo, $showF
         return $message;
     }
 
+    /**
+     * Describes a button recipe change
+     *
+     * @return string
+     */
+    protected function friendly_message_button_recipe_change() {
+        $message .= 'The button recipe for ' . $this->params['buttonName'] . ' changed' .
```

Comment:
Here's your bug, shadowshade - the function starts with `$message .=`

Reply:
Ah, gotcha! Okay, I'll fix that now.

Reply:
Interesting that this doesn't fail on my local rig. Maybe there are differences because of the PHP settings and versions.